<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalEditDialog</class>
 <widget class="QDialog" name="JournalEditDialog">
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextEdit" name="summaryEdit"/>
     </item>
     <item>
      <widget class="QMaemo5ValueButton" name="calendarButton">
       <property name="text">
        <string>Calendar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <enum>QDialogButtonBox::Cancel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
